src/month-days.tsx

```diff
@@ -95,7 +96,7 @@ export function CalendarDays() {
           <Day
             variant={variant}
             day={day}
-            key={format(day, 'd-M')}
+            key={format(day, 'd-M', { locale })}
```

Comment:
**question**: do we need to add the locale here?

Reply:
@uselessdev since it's used for only React key here, it seems unnecessary to pass in the `locale` object. I'll revert this